bundles/aws/build.gradle.kts

```diff
@@ -74,5 +74,5 @@ tasks.jar {
 }
 
 tasks.compileJava {
-  dependsOn(":catalogs:catalog-hadoop:runtimeJars")
+  dependsOn(":catalogs:catalog-fileset:runtimeJars")
```

Comment:
Are we going to rename the package name `org.apache.gravitino.catalog.hadoop` in `hadoop-common`, `xxxx-bundle`?

Reply:
No, I don't want to, because `hadoop-common` will also be used by the clients like gvfs, the changes of package name will break the compatibility.